requirements.txt

```
@@ -4,3 +4,4 @@ netaddr
 psutil
 rfc5424-logging-handler
 yara-python
+wmi
```

```suggestion
wmi
yara-python
```
Please keep requirements sorted alphabetically.  It is not so important when lists are small but when lists grow, it helps avoid duplicate entries and makes it easier to spot missing requirements.